extensions/ql-vscode/package.json

```diff
@@ -164,7 +164,6 @@
       {
         "type": "object",
         "title": "CLI",
-        "id": "codeQL.cli",
```

Comment:
This commit introduces a number of unintentional version downgrades and removes a bunch of JSON fragments that should still be there. Please clean up the git history to only introduce the minimal changes to package.json (upgrading react/react-dom).